Tasks/Common/nuget-task-common/NuGetXmlHelper.ts

```diff
@@ -0,0 +1,128 @@
+import * as fs from "fs";
+import * as ltx from "ltx";
+
+import * as tl from "vsts-task-lib";
+
+import { INuGetXmlHelper } from "./INuGetXmlHelper";
+
+export class NuGetXmlHelper implements INuGetXmlHelper {
+    constructor(private _nugetConfigPath: string) {
+    }
+
+    public SetApiKeyInNuGetConfig(source: string, apiKey: string): void {
+        tl.debug('Setting ApiKey is not supported without using nuget.exe');
```

Comment:
You mentioned earlier this should probably be higher priority message than debug.